econml/tests/test_drlearner.py

```diff
@@ -808,6 +808,27 @@ def fit(self, X, y):
         with pytest.raises(Exception):
             est.fit(y, t, W=w, groups=groups)
 
+    def test_score(self):
```

Comment:
I think it would be better to test all 8 combinations of { multi-class, non-multi-class }×{  y is a vector, y is an (n,1) array }×{ final model always returns a vector, final model always returns an (n,1) array }.  For example, my original bug doesn't cover both multi-class and non-multi-class, but shows that 2 of the remaining 4 scenarios were broken.

Reply:
done